web/js/dashboard.js

```diff
@@ -1,276 +1,276 @@
 $(function(){
+    if (!$('html').is('.ie9')) {
```

Comment:
Rather than indenting the entire contents of this `$(function(){` block, how about just bailing out early? Like…

```javascript
$(function(){
    if ( $('html').is('.ie9') ) {
        return;
    }

    Chart.defaults.global.defaultFontSize = 16;
    …
```